client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -1,9 +1,10 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
 @import '~@wordpress/base-styles/colors';
 
 $design-selector-grid-gap: 4.5em;
 
 $design-selector-max-width: 1100px;
-$design-selector-selection-space: 285px;
+$design-selector-selection-space: 285px + $header-height;
```

Comment:
I don't quite understand the styling changes in here -- are the required for the tabbing fix? :slightly_smiling_face:

Reply:
`$design-selector-selection-space` refers to the space that allows you to peek through the design selector UI when you are on page selection.

With the page selector modal being a child element of the design selector, the selection space has to take account of the header height, otherwise, the content of the design selector becomes clipped (see screenshot).

![image](https://user-images.githubusercontent.com/1287077/74738661-b0cbcc80-5257-11ea-9f71-866146f85e97.png)